Source/WebCore/rendering/ImageQualityController.cpp

```diff
@@ -169,16 +171,16 @@ InterpolationQuality ImageQualityController::chooseInterpolationQuality(Graphics
 
     // If an animated resize is active, paint in low quality and kick the timer ahead.
     if (m_animatedResizeIsActive) {
-        set(object, innerMap, layer, size);
+        set(object, innerMap, layer, scaledLayoutSize);
         restartTimer();
         return InterpolationQuality::Low;
     }
     // If this is the first time resizing this image, or its size is the
     // same as the last resize, draw at high res, but record the paint
     // size and set the timer.
-    if (isFirstResize || oldSize == size) {
+    if (isFirstResize || oldSize == scaledLayoutSize) {
```

Comment:
If these are FloatSizes, I think we need to use `areEssentiallyEqual()` instead of using the `==operator`.

Reply:
`if (isFirstResize || WTF::areEssentiallyEqual(oldSize, scaledDestinationSize))` something like this?

Reply:
Yes. Or you can just say `areEssentiallyEqual()` without the qualifier `WTF`.

Reply:
Done